src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -301,7 +301,7 @@ function FilterSidebarSearchTab({
             className={classes.font}
             style={{
                 boxShadow: 'none',
-                backgroundColor: COLOURS.GREEN,
+                backgroundColor: COLOURS.HEADER,
```

Comment:
Good idea to switch this to a "class-style" name rather than a literal color name however we are now using a color called `HEADER` outside of the header. What do you think about a name like `PRIMARY` or `NAVIGATION` instead?